ucm2/MediaTek/mt8390-evk/HiFi.conf

```diff
@@ -0,0 +1,217 @@
+		cset "name='O053 I027 Switch' on"
+		cset "name='O054 I028 Switch' on"
+		cset "name='O055 I029 Switch' on"
+		cset "name='O176 I070 Switch' on"
+		cset "name='O177 I071 Switch' on"
+	]
+
+	DisableSequence [
+		cset "name='HDMI_OUT_MUX' Disconnect"
+		cset "name='DPTX_OUT_MUX' Disconnect"
+		cset "name='O000 I000 Switch' off"
+		cset "name='O001 I001 Switch' off"
+		cset "name='O002 I004 Switch' off"
+		cset "name='O003 I006 Switch' off"
+		cset "name='O034 I168 Switch' off"
+		cset "name='O035 I169 Switch' off"
+		cset "name='O036 I012 Switch' off"
+		cset "name='O037 I013 Switch' off"
+		cset "name='O040 I002 Switch' off"
+		cset "name='O041 I003 Switch' off"
+		cset "name='O048 I022 Switch' off"
+		cset "name='O049 I023 Switch' off"
+		cset "name='O050 I024 Switch' off"
+		cset "name='O051 I025 Switch' off"
+		cset "name='O052 I026 Switch' off"
+		cset "name='O053 I027 Switch' off"
+		cset "name='O054 I028 Switch' off"
+		cset "name='O055 I029 Switch' off"
+		cset "name='O176 I070 Switch' off"
+		cset "name='O177 I071 Switch' off"
+	]
+}
+
+SectionDevice."Line4" {
+	Comment "I2S output"
+
+	Value {
+		PlaybackPriority 100
+		PlaybackChannels 8
+		PlaybackPCM "hw:${CardId},6"
+	}
+}
+
+SectionDevice."Line2" {
+	Comment "PCM output"
+
+	Value {
+		PlaybackPriority 200
+		PlaybackChannels 2
+		PlaybackPCM "hw:${CardId},2"
+	}
+}
+
+SectionDevice."HDMI" {
+	Comment "HDMI/DP output"
+
+	Value {
+		PlaybackPriority 300
+		PlaybackPCM "hw:${CardId},5"
+		JackControl "HDMI Jack"
+	}
+
+	EnableSequence [
+		cset "name='HDMI_OUT_MUX' 1"
+	]
+
+	DisableSequence [
+		cset "name='HDMI_OUT_MUX' 0"
+	]
+}
+
+SectionDevice."DP" {
+	Comment "HDMI/DP output"
+
+	Value {
+		PlaybackPriority 300
+		PlaybackPCM "hw:${CardId},5"
+		JackControl "DP Jack"
+	}
+
+	EnableSequence [
+		cset "name='DPTX_OUT_MUX' 1"
+	]
+
+	DisableSequence [
+		cset "name='DPTX_OUT_MUX' 0"
+	]
+}
+
+SectionDevice."Speaker" {
+	Comment "Lineout speaker"
+
+	ConflictingDevice [
+		"Headphones"
+	]
+
+	EnableSequence [
+		cset "name='LOL Mux' Playback_L_DAC"
+	]
+
+	DisableSequence [
+		cset "name='LOL Mux' Open"
+	]
+
+	Value {
+		PlaybackPriority 400
+		PlaybackChannels 2
+		PlaybackPCM "hw:${CardId},0"
+	}
+}
+
+SectionDevice."Headphones" {
+	Comment "Earphone speaker"
+
+	ConflictingDevice [
+		"Speaker"
+	]
+
+	EnableSequence [
+		cset "name='HP Mux' Audio Playback"
+	]
+
+	DisableSequence [
+		cset "name='HP Mux' Open"
+	]
+
+	Value {
+		PlaybackPriority 500
+		PlaybackChannels 2
+		PlaybackPCM "hw:${CardId},0"
+	}
+}
+
+SectionDevice."Line3" {
+	Comment "I2S input"
+
+	Value {
+		CapturePriority 100
+		CaptureChannels 2
+		CapturePCM "hw:${CardId},11"
+	}
+}
+
+SectionDevice."Line1" {
+	Comment "PCM input"
+
+	Value {
+		CapturePriority 200
+		CaptureChannels 2
+		CapturePCM "hw:${CardId},8"
+	}
+}
+
+SectionDevice."Mic2" {
+	Comment "Dmic"
+
+	Value {
+		CapturePriority 300
+		CaptureChannels 2
+		CapturePCM "hw:${CardId},14"
+	}
+}
+
+SectionDevice."Mic1" {
+	Comment "Amic"
```

Comment:
Better name ? Analog microphone ? This is string is for users.

Reply:
Done